connector/connect/README.md

```diff
@@ -21,6 +21,28 @@ user experience across all languages. Please follow the below guidelines:
 
 Python-specific development guidelines are located in [python/docs/source/development/testing.rst](https://github.com/apache/spark/blob/master/python/docs/source/development/testing.rst) that is published at [Development tab](https://spark.apache.org/docs/latest/api/python/development/index.html) in PySpark documentation.
 
+To generate the Python client code from the proto files:
+
+First, make sure to have a Python environment with the installed dependencies.
+Specifically, install `black` and dependencies from the "Spark Connect python proto generation plugin (optional)" section.
+
+
+```
+pip install -r dev/requirements.txt
+```
+
+Install [buf](https://github.com/bufbuild/buf)
+
+```
+brew install bufbuild/buf/buf
+```
+
+Generate the Python files by running:
+
+```
+dev/connect-gen-protos.sh
```

Comment:
Don't know is it specific to me but this command fails on my site with error:
```
+ buf generate --debug -vvv
...
Failure: 403 Forbidden
```

A workaround is to fork https://github.com/HyukjinKwon/my-github-actions and run the workflow:
![Screen Shot 2023-03-24 at 8 38 18 PM](https://user-images.githubusercontent.com/1580697/235881720-aedf17a5-b14e-4a62-ab21-576a2221fedf.png)
and download the results:
![Screen Shot 2023-03-24 at 8 38 45 PM](https://user-images.githubusercontent.com/1580697/235881807-1a2a8c34-44bd-44cc-9b5c-1ff11d1eebd0.png)